VTCare/client/src/components/PatientHistory.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <Transition>
+    <div class="modal-mask">
+      <div class="modal-wrapper">
+        <div class="modal-container">
+          <div class="info-container">
+            <div class="modal-header">
+              <a style="color:black"><i class="fa-solid fa-xmark fa-2x" @click="closeModal()"></i></a>
+            </div>
+            <div class="history-header-body">
+              <div class="patient-info">
+                {{ patientName }}'s History
+              </div>
+            </div>
+          </div>
+          <div class="history-container">
+              <div v-for="item in patientHist" :key="item" class="history-item">
+
+                  <p class="apptDate">
+                    <b>On </b>{{formatDate(item.apptDate)}} with Dr. {{item.providerName}}
+                  </p>
+                  <p>
+                    <b>Diagnosis:</b> {{item.diagnosis}}
+                  </p>
+                  <p>
+                    <b>Prescription:</b> {{item.prescription}}
+                  </p>
+                  <p>
+                    <b>Notes:</b> {{item.notes}}
+                  </p>
+
+              </div>
+          </div>
+
+      </div>
+    </div>
+    </div>
+  </Transition>
+</template>
+
+<script lang="ts">
+import patient from "@/services/patient.ts";
+import makeToast from './toast/makeToast';
+import {PatientVisitHistory} from '@/types'
+import moment from 'moment';
+
+export default {
```

Comment:
wrap the exported object around DefineComponent(....) otherwise it will break production.